### ISSUE TITLE
:wheelchair: a11y(bal-modal, bal-popup): Overlay background accessible

### ISSUE THREAD

**Issue by carolabes** [baloise/design-system#1423]
### Description of this issue

* When the menus are expanded, users can access "hidden" content behind overlay via screenreader/keyboard.
* affects all overlays

### How to fix it

* If menus are expanded, only visible content should be accessible
* Background must have tab-index = -1 and aria-hidden = true.

### Additional Information

Level A
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html

Level AA
https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum.html

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mmarinkov** [comment]
on hold - we will work on this task in pair

**Comment by mmarinkov** [comment]
we have to add property in bal-popup not bal-popover
